etc/resampling.yaml

```diff
@@ -12,7 +12,7 @@ resampling:
     resampler: ewa
     kwargs:
       weight_delta_max: 40.0
-      weight_distance_max: 1.0
+      weight_distance_max: 2.0
```

Comment:
This is very confusing. This is already the value in master isn't it? https://github.com/ssec/polar2grid/blob/f21df1604e317d26ccf544905829c93fccc10b53/etc/resampling.yaml#L15

Reply:
Oh I think your master branch is behind the ssec/polar2grid master. Maybe that's why.